runtime/doc/news.txt

```diff
@@ -69,7 +69,8 @@ LSP
 
 LUA
 
-• TODO
+• Empty dict are converted to empty Lua table instead of a "special" typed
```

Comment:
"API functions consistently return an empty dictionary as |vim.empty_dict()|. Before a |lua-special-tbl| was sometimes used. "

Reply:
Thanks, that sounds so much better than the original statement. Done in https://github.com/neovim/neovim/pull/29659/commits/970a27927eb31bdc735f0d7d5e3d07784486c6de